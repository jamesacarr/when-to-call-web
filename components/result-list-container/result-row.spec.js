/* eslint-disable camelcase */

import React from 'react';
import { shallow } from 'enzyme';
import ResultRow from './result-row';

const result = {
  primary: 'Some stuff',
  secondary: 'More stuff'
};

describe('<ResultRow/>', () => {
  describe('.render', () => {
    it('renders correctly', () => {
      const wrapper = shallow(<ResultRow result={result}/>);
      expect(wrapper).toMatchSnapshot();
    });

    it('renders with divider', () => {
      const wrapper = shallow(<ResultRow divider result={result}/>);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
