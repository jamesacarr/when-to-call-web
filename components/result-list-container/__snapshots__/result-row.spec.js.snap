// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultRow/> renders correctly 1`] = `
<WithStyles(ListItem)
  button={true}
  dense={true}
  divider={false}
>
  <WithStyles(ListItemText)
    primary="Some stuff"
    secondary="More stuff"
  />
</WithStyles(ListItem)>
`;

exports[`<ResultRow/> renders with divider 1`] = `
<WithStyles(ListItem)
  button={true}
  dense={true}
  divider={true}
>
  <WithStyles(ListItemText)
    primary="Some stuff"
    secondary="More stuff"
  />
</WithStyles(ListItem)>
`;
